### PR TITLE
Product update

### PR DESCRIPTION
**Pull request by tiennguyen98**

### CODE REVIEW

**Comment by tiennguyen98** [comment]
ready

File: app/Models/Image.php [hunk 1]
```diff
@@ -15,4 +16,10 @@ public function product()
     {
         return $this->belongsTo('App\Product');
     }
+
+    public function delete()
+    {
+        parent::delete();
+        Storage::delete('public/'.$this->file_name);
```
Comment (BunnyPi04) [inline app/Models/Image.php]: convention

File: routes/web.php [hunk 2]
```diff
@@ -129,6 +129,11 @@ function () {
         Route::get('/search', 'ProductController@search')->name('search');
         Route::get('/results', 'ProductController@result')->name('search_results');
         Route::get('/province', 'ProductController@getSearchProvince')->name('get_search_rovince');
+        Route::get('/province', 'ProductController@getSearchProvince')->name('getSearchProvince');
```
Comment (BunnyPi04) [inline routes/web.php]: sao 2 dòng y nhau vậy?
các route name xem lại cách đặt tên route name nhé
https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4

**Comment by BunnyPi04** [comment]
commented

**Comment by tiennguyen98** [comment]
ready

**Pull request merged**